web/themes/custom/kitrab/src/scss/_mainpage.scss

```
@@ -0,0 +1,6 @@
+.layout__region.layout__region--first {
+    .block {
+      margin-right:rem(20);
+      margin-left:rem(20);
+    }
+}
```

Need a blank space at the end of the file.
```suggestion
}

```